packages/core/src/ui/Comments/Utterances.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useCallback } from "react";
+import { useTheme } from "next-themes";
+
+export interface UtterancesConfig {
+  provider: "utterances";
+  pages?: Array<string>;
+  config: {
+    theme?: string;
+    repo?: string;
+    label?: string;
+    issueTerm?: string;
```

Comment:
I'm guessing `theme` and `issueTerm` could be of more strict types, e.g. `theme: "light" | "dark"`?